test/Microsoft.AspNetCore.Mvc.Core.Test/DependencyInjection/ApplicationModelConventionExtensionsTest.cs

```diff
@@ -13,6 +13,34 @@ namespace Microsoft.Extensions.DependencyInjection
     public class ApplicationModelConventionExtensionsTest
     {
         [Fact]
+        public void DefaultParameterModelConvention_AppliesToAllParametersInApp()
+        {
+            // Arrange
+            var app = new ApplicationModel();
+            app.Controllers.Add(new ControllerModel(typeof(HelloController).GetTypeInfo(), new List<object>()));
+            app.Controllers.Add(new ControllerModel(typeof(WorldController).GetTypeInfo(), new List<object>()));
+
+            var options = new MvcOptions();
+            options.Conventions.Add(new SimpleParameterConvention());
+
+            // Act
+            options.Conventions[0].Apply(app);
+
+            // Assert
+            foreach (var controller in app.Controllers)
+            {
+                foreach (var action in controller.Actions)
+                {
+                    foreach (var parameter in action.Parameters)
+                    {
+                        Assert.Contains("TestProperty", parameter.Properties.Keys);
+                        Assert.Contains("TestValue", parameter.Properties.Values);             
```

Comment:
Thanks! Is it okay if I change the other tests (DefaultActionModelConvention and DefaultControllerModelConvention) to be consistent with this? (Not use `Assert.True` and  follow the `Assert.Single` pattern?)